configure.ac

```diff
@@ -2270,6 +2270,11 @@ fi
        fi
     fi
 
+    AC_PATH_PROG([BINDGEN], [bindgen], [no])
+    if test "x$BINDGEN" = "xno"; then
+       AC_MSG_ERROR([bindgen required])
+    fi
```

Comment:
The idea is that when building from a distribution archive this wouldn't be required. Just like cbindgen.